djangoTrading/settings.py

```diff
@@ -11,21 +11,27 @@
 """
 
 from pathlib import Path
+import environ
+import os
+
+env = environ.Env()
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
-
+environ.Env.read_env(os.path.join(BASE_DIR, '.env'))
 
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-0z=7ybt!dh85py0=id=r=78r#*6*c7y9cd2*gr)7q$*j^d2^bt'
+SECRET_KEY = env('SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = [
+    '*'
+]
```

Comment:
allowed hosts should be also in `.env`
As usual you write in `.env` file something like:
```ini
ALLOWED_HOSTS="localhost 0.0.0.0"
```
Or any needed hosts and then in `settings.py`
```python
ALLOWED_HOSTS = os.environ.get('ALLOWED_HOSTS', 'localhost').split() # usual case
ALLOWED_HOSTS = env('ALLOWED_HOSTS').split() # your case
```